code/modules/cargo/randomstock.dm

```diff
@@ -183,7 +177,6 @@ var/list/global/random_stock_uncommon = list(
 	"fireaxe" = 1,
 	"service" = 2,
 	"robot" = 2,
-	"latexb" = 0.5,
```

Comment:
Remind me, what are these?

Reply:
Those latex balloon gloves. Only interesting feature is that they make a gunshot sound when popped.